pkg/sdkserver/localsdk_test.go

```diff
@@ -374,7 +502,7 @@ func TestSDKConformanceFunctionality(t *testing.T) {
 	setAnnotation := "setannotation"
 	l.gs.ObjectMeta.Uid = exampleUID
 
-	expected := []string{}
+	var expected []string
```

Comment:
Curious why this is better than before.

Reply:
Oh, my IDE gave me a hint, and I clicked it. It's nicer because it doesn't allocate space to an array that is empty when nil will work just the same -- but for a test it's relatively inconsequential.   

I can remove it if you would prefer?

Reply:
No need to change, I think it would be better this way.